js/app.js

```diff
@@ -0,0 +1,152 @@
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'last project',
+    title: 'Data Dashboard Healthcare',
+    image: './image_icons/Img_Placeholder.svg',
+    technologies: ['css', 'Codepen', 'Git & Github'],
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque porro aliquam quod repellat harum distinctio quas voluptates eligendi maiores at facilis asperiores, voluptatem placeat cum nostrum nisi quae, natus neque? Minus expedita, aliquid repellat recusandae laboriosam natus culpa odit.',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+];
+
+const divProject = document.createElement('div');
+
+// add modal function
+function addEvents() {
+  for (let i = 0; i < openPopupButtons.length; i += 1) {
+    openPopupButtons[i].addEventListener('click', (ev) => {
+      const projectId = ev.target.id;
+      divProject.classList.add('popup-overlay-objects');
+      document.querySelector('.popup-bg').style.display = 'flex';
+      // create techs
+      let techsHTML = '';
+      for (let k = 0; k < projects[i].technologies.length; k += 1) {
+        techsHTML += `<li><p>${projects[i].technologies[k]}</p></li>`;
+      }
+      divProject.innerHTML = '<div class="close-btn">+</div>'
+      + `<h3 class="intru-title">${projects[projectId].title}</h3>`
+      + `<ul class="languages">${techsHTML}</ul>`
+      + '<div class="content d-flex flex-dir-col">'
+      + '<div class="content-image">'
+      + `<img src="${projects[projectId].image}" alt="modal-image" />`
+      + '</div>'
+      + '<div class="body d-flex flex-dir-col">'
+      + `<p>${projects[projectId].description}</p>`
+      + '<div class="d-flex pop-buttons">'
+      + '<button class="btn-primary btn-pop d-flex" type="button">'
+      + 'See Live <i class="fa fa-clock-o"></i></button>'
+      + '<button class="btn-primary btn-pop d-flex" type="button">'
+      + 'See Source <i class="fa fa-github"></i></button>'
+      + '</div>'
+      + '</div>'
+      + '</div>';
+
+      document.getElementById('overlay').appendChild(divProject);
+
+      const deleteDetail = document.querySelector('.close-btn');
+      deleteDetail.addEventListener('click', () => {
+        divProject.remove();
+        document.querySelector('.popup-bg').style.display = 'none';
+      });
+    });
+  }
+}
+
+addEvents();
+
+/* + `<h3 class="intru-title">${projects[0].title}</h3>`
++ '<div class="content d-flex flex-dir-col">'
++ '<div class="content-image">'
++ `<img src="${projects[projectId].image}" alt="modal-image" />`
++ '</div>'
++ '<div class="body d-flex flex-dir-col">'
++ `<p>${projects[projectId].description}</p>`
++ '<div class="d-flex pop-buttons">'
++ '<button class="btn-primary btn-pop d-flex" type="button">'
++ 'See Live <i class="fa fa-clock-o"></i></button>'
++ '<button class="btn-primary btn-pop d-flex" type="button">'
++ 'See Source <i class="fa fa-github"></i></button>'
++ '</div>'
++ '</div>'
++ '</div>';
+console.log(project[0]) */
```

Comment:
- Take your project at the next level. Remove unused commented lines.

Reply:
Done thanks for your review

Reply:
Issue well resolved! ✔️